intel_extension_for_transformers/llm/evaluation/lm_eval/evaluator.py

```diff
@@ -164,8 +164,8 @@ def evaluate(model,
         limit=limit,
         bootstrap_iters=bootstrap_iters,
         decontamination_ngrams_path=decontamination_ngrams_path,
-        write_out=write_out,
-        output_base_path=output_base_path
+        # write_out=write_out,

```

Comment:
why comment out?

Reply:
Thanks for catching this. It's a version issues of lm-eval. Let me install the most recent version, validate and update this.

Reply:
validated lm-eval latest commit https://github.com/EleutherAI/lm-evaluation-harness.git@cc9778fbe4fa1a709be2abed9deb6180fd40e7e2.